packages/contracts/contracts/L1/L2OutputOracle.sol

```diff
@@ -117,6 +120,20 @@ contract L2OutputOracle is Ownable {
         emit l2OutputAppended(_l2Output, _l2timestamp);
     }
 
+    /**
+     * @notice Deletes the most recent output.
+     * @param _l2Output The value of the most recent output. Used to prevent erroneously deleting
+     *  the wrong root
+     */
+    function deleteL2Output(bytes32 _l2Output) external onlyOwner {
+        bytes32 outputToDelete = l2Outputs[latestBlockTimestamp];
+        require(_l2Output == outputToDelete, "Can only delete the most recent  output.");
```

Comment:
nit: two spaces between recent and output and also curious why you use `require` here vs a custom error, I have a slight preference for consistency but open for using either when more appropriate

Reply:
I used `require` here simply because that's what was used in the rest of the contract, and I wanted to be consistent internally. That said, this is the only new contract still using `require`. Would you approve if I expand the scope of this PR with a new commit that introduces custom errors?

Reply:
Yes I would! But if it's additional work that you don't want to do, that's cool too